src/include/duckdb/main/capi/header_generation/apis/v1/v1.2/v1.2.0.json

```diff
@@ -357,6 +357,9 @@
         "duckdb_append_varchar",
         "duckdb_append_varchar_length",
         "duckdb_append_blob",
-        "duckdb_append_null"
+        "duckdb_append_null",
```

Comment:
The current approach is that we only stabilize C API functions once per release. Any new functions we add now should go into unstable. 

The idea is that you should in theory be able to grab the duckdb_extension.h file from any version and have it produce valid duckdb extensions. If we start appending to files in `src/include/duckdb/main/capi/header_generation/apis/v1/v1.2/*.json` we break this property.

Reply:
I can update the README for the C headers to properly reflect this

Reply:
Done! :)